code/datums/mood_events/generic_positive_events.dm

```diff
@@ -124,6 +132,26 @@
 	mood_change = 3
 	timeout = 5 MINUTES
 
+/datum/mood_event/chapel_comforted
+	description = "<span class='nicegreen'>You are comforted by the holy aura of the chapel.</span>\n"
+	mood_change = 3
+	timeout = 15 MINUTES
+
+/datum/mood_event/comically_comforted
+	description = "<span class='nicegreen'>You are comforted by the presence of a comical person.</span>\n"
+	mood_change = 3
+	timeout = 5 MINUTES
+
+/datum/mood_event/silently_comforted
+	description = "<span class='nicegreen'>You are comforted by the presence of a silent person.</span>\n"
+	mood_change = 3
+	timeout = 5 MINUTES
+
+/datum/mood_event/theater_comforted
+	description = "<span class='nicegreen'>You are comforted by your time in the theatre.</span>\n"
+	mood_change = 3
+	timeout = 15 MINUTES
```

Comment:
Use area moodlets for this.

Reply:
You mean for the theatre and chapel perks, yeah? The comical and silent comforts are conferred on examine.

Reply:
for the buckle ones yea

Reply:
Would you happen to have an example of where area moodlets are used for reference?

Reply:
bar area code